pieces.py

```diff
@@ -109,17 +106,16 @@ class Pawn(Piece):
                              [5,  10,  10, -20, -20,  10, 10,   5],
                              [0,   0,   0,   0,   0,   0,  0,   0]]
 
-    def __init__(self, is_white, place):
-        if is_white:
-            square = squares[1][place]
-            image = self.WHITE_IMAGE
+    def __init__(self, team: Team, tur=None, square=None):
+
+        if square is None:
+            square = squares[Pawn.LINE_OF_WHITE_PAWNS][tur] if team.is_white_team else squares[Pawn.LINE_OF_BLACK_PAWNS][tur]
+            super().__init__(square, team)
         else:
-            square = squares[6][place]
-            image = self.BLACK_PAWN
-        super().__init__(image, square, is_white, 'Pawn')
+            super().__init__(square, team)
```

Comment:
You don't need an if else block here.
You could just leave the if as it is, and then call super().__init__(square, team).